### ISSUE TITLE
Removing features?

### ISSUE THREAD

**Issue by xet7** [wekan/wekan#3394]
Hi,
it seems UCS is doing [UCS 5.0 – Focus instead of feature overkill](https://www.univention.com/blog-en/2020/12/ucs-5-0-discontinued-features/).

For some, Wekan already has feature overkill.

Everyone, what do you think? What features could be removed from Wekan?

**Comment by daniel-eder** [comment]
While I agree that Wekan offers a lot of features, I do not think at this point they are overkill.
When comparing to other kanban boards/project management software (proprietary and open source) it seems a common theme in this field to have a range of features (automation, customization) that surround the core functionality (kanban board). 

Nobody uses all of these, but each feature usually has a significant user base. 
While some Wekan features are somewhat "hacky" (looking at you, subtasks!) they still fill a valid role for some users and generally don't hurt others. 

TLDR: I think removing features is not necessarily a good move at this point, but I cannot hurt to identify a few core features to focus on for stability, bugfixes and QoL improvements, while making the rest prio 2.

**Comment by helioguardabaxo** [comment]
I usually use most of the Wekan features. These features are very useful.
I think Wekan is a mature solution. If I had the resources to organize boards and teams (#802) it would be perfect.

**Comment by xet7** [comment]
It seems that for some, speed seems more important https://github.com/wekan/wekan/issues/3345#issuecomment-745264163

**Comment by daniel-eder** [comment]
> It seems that for some, speed seems more important [#3345 (comment)](https://github.com/wekan/wekan/issues/3345#issuecomment-745264163)
Definitely a valid concern - the question is which of the current features really impact speed negatively to a measurable degree and *when* each feature has an impact (e.g. I doubt custom fields or board rules [should] have a negative impact on user lookup).

**Comment by xet7** [comment]
@daniel-eder 

At [Wekan survey](https://blog.wekan.team/2020/06/results-of-wekan-survey-2020-01/) is:
- More scalability
- Make performance match Trello
- More stability
- Fix rising CPU load

So it's like, Wekan users would like to:
- Have Wekan work on their single small server as fast as Trello's big optimized cluster
- Add unlimited amount of Rules, Outgoing webhooks etc
- Have about 20 000 to 60 000 simultaneous interacting users
- Wekan implemented with fastest possible way, every bit optimized
- Still have all the same features, and add a lot more

In some point it does anyway hit the limits of disk speed, networks speed, multi-CPU usage etc.

**Comment by xet7** [comment]
Upcoming is also searching from all boards, cards, etc that some Wekan user is participating. That's a nice way to increase database load. And importing huge boards from Trello to Wekan.

**Comment by xet7** [comment]
I did a little tests about minimal Wekan version. It becoming full Wekan version depends is there interest and [funding](https://wekan.team/commercial-support/) for it.

This minimal Wekan version does not have Javascript at frontend. It works on webbrowsers that current full Wekan version does not work yet, like Netsurf and IE. For backend, I'm testing various alternatives.

Netsurf webbrowser:

![wekan_netsurf](https://user-images.githubusercontent.com/15545/102415327-a9ed2200-4000-11eb-8f5c-596c619e0900.png)

IE:

![wekan-ie](https://user-images.githubusercontent.com/15545/102414445-3bf42b00-3fff-11eb-9bf0-dbc8409a1531.png)
